modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts

```diff
@@ -52,14 +67,27 @@ describe('browser.tinymce.themes.silver.editor.color.ColorPickerSanityTest', ()
       const setHexWhite = setHex('ffffff');
       const setHexBlack = setHex('000000');
 
-      const pSubmitDialog = async (editor: Editor) => {
+      const submit = async (editor: Editor) => {
         const docBody = getBody(editor);
         FocusTools.setFocus(docBody, dialogSelector);
         await Waiter.pTryUntil('Button is not disabled', () => UiFinder.notExists(docBody, 'button.tox-button:contains("Save")[disabled]'));
         TinyUiActions.submitDialog(editor);
+        return docBody;
+      };
+
+      const pSubmitDialog = async (editor: Editor) => {
+        const docBody = await submit(editor);
         await Waiter.pTryUntil('Dialog should close', () => UiFinder.notExists(docBody, dialogSelector));
       };
 
+      const pSubmitDialogWithExpectedAlert = async (editor: Editor, expectedAlert: string) => {
+        const docBody = await submit(editor);
+        await Waiter.pTryUntil('Alert should show correct message', () => UiFinder.exists(docBody, `p:contains(${expectedAlert})`));
+        // Close alert
+        TinyUiActions.clickOnUi(editor, 'button[title="OK"]');
```

Comment:
We normally do something like this instead here, as we only use attribute when there is no text.
```suggestion
        TinyUiActions.clickOnUi(editor, 'button:contains("OK")');
```